client-java/src/main/resources/palisade-client2.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# replacement attributes markers:
+#  %t = token
+#  %s = timestamp (yyyyMMdd-hhmmss)
+#  %r = resource
+#
+service:
+  palisade:
+    path: palisade/registerDataRequest
+  filteredResource:
+    path: filteredResource/name/%t
+  data:
+    path: data/read/chunked    
```

Comment:
All of the YAML stuff has been removed. As it's been refactored, the need for it has diminished to the point where it can be removed with no adverse effects. It actually makes it simpler.